datalad/tests/fixtures.py

```diff
@@ -0,0 +1,73 @@
+"""utilities to assist with setting up tests"""
+
+from datalad.api import clone
```

Comment:
Add direct import trio the line below, API is heavyish

Reply:
Agreed.